docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+services:
+  kt-app:
+     image: jakefrosty/korotagger:1.3
```

Comment:
could you add the docker command to generate the image and start docker compose to the readme

Reply:
Will do! I'll make a follow up commit to the README.md file